src/bower_config.ts

```diff
@@ -13,7 +13,11 @@ import * as fs from 'fs';
 
 function bowerConfigPath(root?: string): string {
   root = root || process.cwd();
-  return path.resolve(root, 'bower.json');
+  try {
+    return path.resolve(root, 'bower.json');
```

Comment:
`path.resolve()` is only responsible for resolving file path fragments. It will never throw (see https://nodejs.org/docs/latest/api/path.html#path_path_resolve_path)